pulp_2to3_migration/app/pre_migration.py

```diff
@@ -134,13 +134,13 @@ def delete_removed_pulp2_content(content_model):
         total=total_content))
 
     pulp2content_pb = ProgressReport(
-        message='Pre-migrating Pulp 2 {} content (general info)'.format(content_type.upper()),
+        message='Pre-migrating Pulp 2 {} content (general info)'.format(content_type),
```

Comment:
Instead of dropping upper() here, maybe leave these as they are and 'upper' the message at content:236? I found the UPPER words easier to pick out.

That may just be a me-thing, however, I'm not tied to it!

Reply:
That's funny. The are/were upper() exactly for the readability but I thought it's just my thing and someone was asking to make it lowercase, maybe it was katello :)

Reply:
https://pulp.plan.io/issues/7732

Reply:
Ha! Well, I find the caps make it easier to see what's going on - without them, the response becomes a paragraph, and it's harder for the eye to pick out what's going on. I def prefer "ease of finding the critical info" over "pretty". If I were submitting the PR, I'd leave the caps, but I certainly won't block merging if you decide to go w/ jsherril's suggestion.

Reply:
I think I'd go with lowercase option, just to be on the safe side. Different people can read/interpret caps as something other than just the intention to make it readable.

Reply:
Sounds good to me :)